### ISSUE TITLE
Inner exceptions are not displayed

### ISSUE THREAD

**Issue by mausch** [machine-visualstudio/machine.vstestadapter#11]
When running the following spec:

```C#
It foobar = () =>
{
    throw new Exception("baz", new Exception("inner ex!"));
};
```

the test explorer says:

```
Test Failed - foobar
Message: baz
```

i.e. it's not showing the inner exception. Is this a known issue? I'm using VS2013.

Cheers

**Comment by eugeneduvenage** [comment]
Hi, I have reproduced your issue fairly easily, it's a simple one to fix.

**Comment by ivanz** [comment]
This should be fixed in the just released 2.0.0-beta1
